augly/tests/text_tests/transforms_unit_tests.py

```diff
@@ -41,7 +41,6 @@ def are_equal_metadata(
     return True
 
 
-
```

Comment:
Don’t forget to rename this file, and text_tests/functional_unit_tests.py too! They’re currently not being found by pytest in the action.